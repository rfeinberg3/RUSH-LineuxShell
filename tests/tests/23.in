path /bin /autograder/source/bin
p2.sh > /tmp/output22 & p1.sh > /tmp/output22
cat /tmp/output22
rm -f /tmp/output22
exit
