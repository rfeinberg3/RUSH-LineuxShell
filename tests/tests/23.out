rush> rush> rush> elephant
rush> rush> 